docs/source/ingest/destination_connectors/code/bash/clarifai.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+unstructured-ingest \
+  local \
+  --input-path example-docs/book-war-and-peace-1225p.txt \
+  --output-dir local-output-to-clarifai \
+  --strategy fast \
+  --chunk-elements \
+  --num-processes 2 \
+  --verbose \
```

Comment:
I think it becomes revelatory and useful for new users to see chunking and embedding arguments here and chunking / embedding class instances in the python file example. Here is a [user realizing their existence](https://unstructuredw-kbe4326.slack.com/archives/C044N0YV08G/p1710839945826309?thread_ts=1710831989.266139&cid=C044N0YV08G). 

I can recommend adding chunking & embedding arguments / objects whenever we got a vector store destination going in

Reply:
Totally agree. Unfortunately ClarifAI won't take embedded documents at this time. It provides that service.
From clarifai:
"Yes, we are currently supporting chunked elements, We are not supporting embedded inputs because our clarifai app by default indexes and embeds the inputs using base embedding model, while ingestion and store it in the DB. We may extend our support in future for embeddings but unsure about the timelines."